test/release/examples/benchmarks/shootout/knucleotide.chpl

```diff
@@ -127,7 +127,7 @@ inline proc hash(str, beg, param size) {
 
 proc string.toBytes() {
   var byteArr: [1..this.numBytes] uint(8);
-  for (b, i) in zip(byteArr, 1..) do
+  for (b, i) in zip(byteArr, 0..) do
```

Comment:
I'm torn whether byteArr should also be updated to start at 0.  It'll be a more painful change that way, as I would expect the rest of the file will have to be updated as a result, but it would maintain the symmetry and that seems valuable in the shootouts

Reply:
Symmetry with what?

Reply:
The symmetry between the string indexing and the indexing of the array that is being returned by this method

Reply:
And by valuable you mean "valuable in terms of clarity"?

Reply:
Yeah, it'll be more understandable that way.  It'll also better reflect the language defaults, though one could argue that the array change should go in your upcoming array branch instead - I'd be okay with that, but I think it also makes sense to do it here when the connection between the changes is more obvious